LongestPalindromicSubstring.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def longestPalindrome(self, s: str) -> str:
+        L = R = l = r = 0
+
+        while r < len(s):
+            if l == r:
+                # Looking for the 'root' of palindrome,
+                # either odd or even number of same characters
+                while r + 1 < len(s) and s[l] == s[r+1]:
+                    r += 1
+                    if l > 0 and s[l - 1] == s[r]:
+                        l -= 1
+
+            if 0 <= l and s[l] == s[r]:
+                L, R = (l, r+1) if ((r+1)-l > R-L) else (L, R)
```

Comment:
It's good to know such things are possible but it'd be better to have two lines for better understanding, like:

```
L = max(L, l)
R = max(R, r + 1)
```

Reply:
Ye, not readable clearly. I made it better, wait for PR